h5netcdf/tests/test_h5netcdf.py

```diff
@@ -352,9 +366,12 @@ def read_legacy_netcdf(tmp_netcdf, read_module, write_module):
     assert v.dimensions == ()
     assert v.ncattrs() == []
 
-    v = ds.variables["var_len_str"]
-    assert v.dtype == str
-    assert v[0] == _vlen_string
+    if backend == "pyfive":
+        warnings.warn("pyfive tests ignore var_len_str")
```

Comment:
I would omit these warnings from the test suite, I don't think add much useful information.